src/main/scala/edu/ie3/simona/model/participant/load/profile/LoadProfileStore.scala

```diff
@@ -215,15 +219,26 @@ object LoadProfileStore extends LazyLogging {
                   .map(typeDay => dynamization(typeDay.getMaxValue, 366))
                   .getOrElse(0d)
               })
-              .max
+              .maxOption
+              .getOrElse(
+                // this should not happen - there's more than one DayType
+                throw new RuntimeException(
+                  "Maximum load not found."
+                )
+              )
```

Comment:
If this cannot happen, avoid the exception. This means, please refactor this part such that a mapping is only produced if maxConsuption is available. Something like the following pseudo code should do the trick: 

```
knownLoadProfiles
      .flatMap(loadProfile => {
       loadProfile match {
         case BdewLoadProfile.H0 => 
            ....maxOption
        }.map(maxConsumption => loadProfile -> maxConsumption)
      }).toMap
```